app/models/event.rb

```diff
@@ -1,5 +1,5 @@
 class Event < ApplicationRecord
   belongs_to :user
   has_many :requests
-  has_many :users, through: :requests
```

Comment:
Why don't we need to specify `through: :requests` in this case? The event does not have direct access to many users because of n:n relationship between the two models.